Add missing <sys/select.h> include

This header is needed when types like fd_set are used. This fixes a
build issue occuring on the musl C library.

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
[Thomas: better commit log.]
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/mpcenc/mpcenc.h
===================================================================
--- a/mpcenc/mpcenc.h
+++ b/mpcenc/mpcenc.h
@@ -34,6 +34,7 @@
 //// portable system includes //////////////////////////////////////
 #include <stddef.h>
 #include <math.h>
+#include <sys/select.h>
 
 //// system dependent system includes //////////////////////////////
 // low level I/O, where are prototypes and constants?
